db/db_range_del_test.cc

```diff
@@ -809,7 +809,7 @@ TEST_F(DBRangeDelTest, TailingIteratorRangeTombstoneUnsupported) {
       // For L1+, iterators over files are created on-demand, so need seek
       iter->SeekToFirst();
     }
-    ASSERT_TRUE(iter->status().IsNotSupported());
+    ASSERT_TRUE(iter->status().IsNotSupported()) << i;
```

Comment:
Removing.

(It prints the value of `i` if the assertion fails. I added it for debugging when this test was failing, then left it because there's a small probability that it may come in handy for someone else later. Removing it, just to avoid touching an extra file without a good reason.)